src/Altinn.App.Api/Controllers/StatelessDataController.cs

```diff
@@ -115,12 +115,24 @@ public class StatelessDataController : ControllerBase
             // runs prefill from repo configuration if config exists
             await _prefillService.PrefillDataModel(owner.PartyId, dataType, appModel);
 
-            Instance virutalInstance = new Instance() { InstanceOwner = owner };
-            await _dataProcessor.ProcessDataRead(virutalInstance, null, appModel);
+            Instance virtualInstance = new Instance() { InstanceOwner = owner };
+            await ProcessAllDataWrite(virtualInstance, appModel);
```

Comment:
@tjololo This was not good!!!

Happens to be OK, as `ProcessAllDataWrite` actually calls `processDataRead` 😊 ☺️ I'll fix in the next rewrite that uses json patch